### ISSUE TITLE
How many GPU VRAM do I need if I use the clip-flant5-xl model？

### ISSUE THREAD

**Issue by ywdong** [linzhiqiu/t2v_metrics#5]
How many GPU VRAM do I need if I use the clip-flant5-xl model？Thx!

**Comment by linzhiqiu** [comment]
`clip-flant5-xxl`: 22394MiB for model loading
`clip-flant5-xl`: 6900MiB for model loading

To run these models on image-text pairs, you will need some additional VRAM (like 500-3000MiB) depending on the batch size. You can fix it to 1 image-text pair if you are working with smaller GPUs. You can also use `with torch.no_grad():` to avoid storing the gradient.
